root/usr/share/nethesis/NethServer/Template/Getmail/Modify.php

```diff
@@ -16,6 +16,7 @@
     ->insert($view->textInput('Server'))
     ->insert($view->textInput('Username'))
     ->insert($view->textInput('Password'))
+    ->insert($view->textInput('Mailbox'))
```

Comment:
If more than one element is allowed, we could use a ``textArea()``, as previous code example does.